backend/src/main/java/com/zeros/backend/domain/services/impl/SimpleRestaurantSubscriber.java

```diff
@@ -22,13 +22,12 @@ public SimpleRestaurantSubscriber(RestaurantRepository restaurantRepository) {
     @Override
     public Restaurant subscribe(Restaurant restaurant) {
 
-        Preconditions.checkArgument(!Strings.isNullOrEmpty(restaurant.getName()) ,"Restaurant name should be present");
-        Preconditions.checkArgument(!Strings.isNullOrEmpty(restaurant.getaddress()),"Restaurant address should be present");
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(restaurant.getName()), "Restaurant name should be present");
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(restaurant.getaddress()), "Restaurant address should be present");
 
```

Comment:
@zeroual is it intellij code formatter ?

Reply:
yes, we need to have the same formater :(